app/cdap/components/Connections/Create/index.tsx

```diff
@@ -69,10 +69,13 @@ export function CreateConnection({
   const [loading, setLoading] = React.useState(true);
   const [state, dispatch] = React.useReducer(reducer, initialState);
   const activeCategory = React.useRef(null);
-  const [connectionDetails, setConnectionDetails] = React.useState<IConnectorDetails>({
+  const [connectionDetails, setConnectionDetails] = React.useState<
```

Comment:
If you modify `React.useState` please replace it with `useState` and `import {useState} from react` in the future.

Reply:
Is this tech debt? Should I just modify all while I'm at it?

Reply:
It definitely is tech debt - i usually modify any instances of this when i see it and am modifying that file because theres no way to screw it up and its very easy to do. 

Basically i look at it like there are two types of code tech debt: 
- Small pieces of tech debt that should be changed any time you see them _and_ modify the file like `react.useState` and `<If />`
- Larger pieces like converting from class components or react.createClass components to functional + hooks should only be done if you're heavily modifying the file (use your judgement here).